vars/onTerraformChangeInPR.groovy

```diff
@@ -2,14 +2,16 @@ def call(Closure block) {
   folderExists('infrastructure') {
     writeFile file: 'check-infrastructure-files-changed.sh', text: libraryResource('uk/gov/hmcts/infrastructure/check-infrastructure-files-changed.sh')
 
-    def infraFolderHasChanges = sh(
-      script: "chmod +x check-infrastructure-files-changed.sh\n" +
-        "    ./check-infrastructure-files-changed.sh",
-      returnStatus: true
-    )
-    sh 'rm check-infrastructure-files-changed.sh'
-    if (infraFolderHasChanges == 1) {
-      return block.call()
+    withCredentials([usernamePassword(credentialsId: credentialsId, passwordVariable: 'BEARER_TOKEN', usernameVariable: 'APP_ID')]) {
+      def infraFolderHasChanges = sh(
+        script: "chmod +x check-infrastructure-files-changed.sh\n" +
+          "    ./check-infrastructure-files-changed.sh",
+        returnStatus: true
+      )
+      sh 'rm check-infrastructure-files-changed.sh'
+      if (infraFolderHasChanges == 1) {
```

Comment:
can you add a bit more printf statements like here https://github.com/hmcts/cnp-jenkins-library/pull/1263/files

Reply:
The checks have passed on this one @charlesluokb ?

Reply:
Is that not abit excessive with the logging?

Reply:
I'm happy to remove now it's debugged and can see it working, @charlesluokb did you just want these for debugging purpose?

Reply:
I removed them, we can add in future if they're needed

Reply:
ok. I found the logging was useful to me when I was investigating the issue as I can see the error status codes (>1) as well as understand what the check is for. generally speaking `0` and `1` mean false and true respectively, here they mean 'changes not found`, 'changes found` , and errors i.e. >1